src/utp_wrapper.rs

```diff
@@ -47,12 +48,20 @@ impl UtpWrapper {
                                     }
                                 },
                                 Err(TryRecvError::Disconnected) => break 'outer,
-                                Err(TryRecvError::Empty) => break,
+                                Err(TryRecvError::Empty) => {
+                                    attempts += 1;
+                                    if attempts < 5000 {
+                                        break;
+                                    } else {
+                                        break 'outer;
```

Comment:
It's okay to the `orx` to be `Empty`. It happens when the crust user hasn't scheduled any message to be written. I'm not understanding why this should trigger a connection loss.